__tests__/articles.test.js

```diff
@@ -84,3 +84,34 @@ describe("/api/articles/:article_id", () => {
       .expect({ msg: "ID must be an integer" });
   });
 });
+
+describe("/api/articles/:article_id/comments", () => {
+  test("GET:200 sends all comments by Article ID to the client", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.comments.length).toBe(11);
+        response.body.comments.forEach((comment) => {
+          expect(typeof comment.article_id).toBe("number");
+          expect(typeof comment.comment_id).toBe("number");
+          expect(typeof comment.author).toBe("string");
+          expect(typeof comment.body).toBe("string");
+          expect(typeof comment.votes).toBe("number");
+        });
+      });
+  });
+  test("GET: 404 returns not found if article does not exist", () => {
+    return request(app)
+      .get("/api/articles/9999/comments")
+      .expect(404)
+      .expect({ msg: "Unable to find Article by ID - 9999" });
+  });
```

Comment:
One test missing here: 400 invalid article id EG /api/articles/not-a-number/comments